dbms/src/Flash/Mpp/ExchangeReceiver.cpp

```diff
@@ -498,6 +360,26 @@ ExchangeReceiverBase<RPCContext>::~ExchangeReceiverBase()
     }
 }
 
+template <typename RPCContext>
+void ExchangeReceiverBase<RPCContext>::waitAllConnectionDone()
+{
+    std::unique_lock lock(mu);
+    auto pred = [&] {
+        return live_connections == 0;
+    };
+    cv.wait(lock, pred);
```

Comment:
And then we should call `cv.notify()` in `ExchangeReceiverBase<RPCContext>::connectionDone`

Reply:
> And then we should call `cv.notify()` in `ExchangeReceiverBase<RPCContext>::connectionDone`

okk